<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coast_car_round_trip_Window</class>
 <widget class="QMainWindow" name="coast_car_round_trip_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="coast_car_round_trip_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro Black</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Вычисление стоимости поездки на автомобиле на дачу (туда и обратно).</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="result_frame">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Результат</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="result_text">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>841</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro Black</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2(потребление бензина на 100 км. пути / 100 × расстояние до дачи × цена одного литра бензина)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="input_frame">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Gas_km_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Gas100km_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Gas100km</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Distance_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Distance_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Distance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="gas_price_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="gas_price_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Price1Lgas</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="coast_car_round_trip_result_button">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
